config/splat.us.stwrp.yaml

```diff
@@ -101,5 +101,5 @@ segments:
       - [0x12E54, c, e_room_fg]
       - [0x12F40, c, popup]
       - [0x13270, c, prim_helpers]
-      - [0x13AA0, sbss] # bss?
+      - [0x13AA0, c, data]
```

Comment:
I checked locally and this seems to work and is probably more correct? From what I remember this didn't use to work due to the linker complaining about .common or something. @Xeeynamo what do you think?
```
      - [0x13AA0, .bss, data]
```

Reply:
Changed.

I had originally set it to `.bss` but my definitions included assignments so this was failing. It seems to work correctly now that they're actually BSS.